vulnerabilities/api.py

```diff
@@ -46,11 +46,29 @@ class MinimalPackageSerializer(serializers.HyperlinkedModelSerializer):
     Used for nesting inside vulnerability focused APIs.
     """
 
+    affected_by_vulnerabilities = serializers.SerializerMethodField("get_affected_vulnerabilities")
+
+    def get_affected_vulnerabilities(self, package):
+        parent_affected_vulnerabilities = package.fixed_package_details.get("vulnerabilities", None)
+        affected_vulnerabilities = []
+
+        if parent_affected_vulnerabilities:
+            for vuln in parent_affected_vulnerabilities:
+                affected_vulnerability = {}
+
+                affected_vulnerability["vulnerability"] = vuln.get(
+                    "vulnerability", None
+                ).vulnerability_id
+
+                affected_vulnerabilities.append(affected_vulnerability)
+
+        return affected_vulnerabilities
+
     purl = serializers.CharField(source="package_url")
 
     class Meta:
         model = Package
-        fields = ["url", "purl", "is_vulnerable"]
```

Comment:
we should not remove ``is_vulnerable``, this will be a breaking change, we should support this variable for now

Reply:
Agreed and restored (along with much work re how to treat and test "lesser" fixed by package versions ;-) and thank you for reminding me of this important principle @TG1999 .  👍